src/containers/Review/ReviewPageWrapper.tsx

```diff
@@ -17,6 +20,24 @@ const ReviewPageWrapper: React.FC = () => {
     serialNumber,
   })
 
+  const [updateReviewResponse] = useUpdateReviewResponseMutation({
+    onCompleted: ({ updateReviewResponse }) =>
+      console.log('Success to update reviewResponse: ', updateReviewResponse?.clientMutationId),
+    onError: (error) => console.log('Problem to update reviewResponse', error.message),
+    refetchQueries: [
+      {
+        query: getReviewQuery,
+        variables: { reviewId: Number(reviewId), serialNumber },
+      },
+    ],
+  })
+
+  const updateResponses = (array: ReviewQuestionDecision[]) => {
```

Comment:
```suggestion
  const updateResponses = async(array: ReviewQuestionDecision[]) => {
```
If you make this async, then each `forEach` mutation will fire off in parallel, which is probably a good thing in this case?

Reply:
Yes, I thought that would already do it, since it doesn't wait for the result. But I'll add this just in case.